src/components/grid/grid-container/grid-container.component.js

```diff
@@ -5,17 +5,62 @@ import GridItem from '../grid-item';
 
 const GridContainer = (props) => {
   const {
-    children
+    children,
+    m,
+    ml,
+    mr,
+    mt,
+    mb,
+    mx,
+    my,
+    gridGap
   } = props;
+
+  const styledSystemProps = {
+    m,
+    ml,
+    mr,
+    mt,
+    mb,
+    gridGap
+  };
+
+  if (mx) {
+    styledSystemProps.mx = mx;
+  }
+
+  if (my) {
+    styledSystemProps.my = my;
+  }
+
   return (
-    <GridContainerStyle data-component='grid'>
+    <GridContainerStyle
+      data-component='grid'
+      { ...styledSystemProps }
+    >
       {children}
     </GridContainerStyle>
   );
 };
 
 GridContainer.propTypes = {
   /** Defines the Components to be rendered within the GridContainer. Requires a GridItem */
-  children: PropTypes.oneOfType([GridItem, PropTypes.arrayOf(GridItem)]).isRequired
+  children: PropTypes.oneOfType([GridItem, PropTypes.arrayOf(GridItem)]).isRequired,
+  /** Any valid CSS value to override default margin */
```

Comment:
I think these prop descriptions need to reflect the fact they can be a number, multiplier of the base spacing constant 8px